pipelines/internal/commands/run/run.go

```diff
@@ -499,7 +510,7 @@ func addRequiredDisks(pipeline *genomics.Pipeline) {
 func addRequiredScopes(pipeline *genomics.Pipeline) {
 	scopes := &pipeline.Resources.VirtualMachine.ServiceAccount.Scopes
 	for _, action := range pipeline.Actions {
-		if strings.HasPrefix(action.ImageUri, "google/cloud-sdk") || isCloudCommand(action.Commands[0]) {
+		if strings.HasPrefix(action.ImageUri, "google/cloud-sdk") || (action.Commands != nil && isCloudCommand(action.Commands[0])) {
```

Comment:
prefer len(x) > 0 to x != nil for slice emptiness

this should go in a separate change since it looks like it's a bug fix